examples/terraform-aws-network-example/variables.tf

```diff
@@ -0,0 +1,34 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Can you update the variables to use `_` instead of `-`? This works better with syntax highlighting, and is more consistent with terraform (e.g terraform uses `aws_instance` instead of `aws-instance`).

Reply:
Same with the outputs.